dbms/src/Interpreters/Settings.h

```diff
@@ -291,7 +291,10 @@ struct Settings
     M(SettingBool, enable_pipeline, false, "Enable pipeline model")                                                                                                                                                                     \
     M(SettingUInt64, pipeline_cpu_task_thread_pool_size, 0, "The size of cpu task thread pool. 0 means using number_of_logical_cpu_cores.")                                                                                             \
     M(SettingUInt64, pipeline_io_task_thread_pool_size, 0, "The size of io task thread pool. 0 means using number_of_logical_cpu_cores.")                                                                                               \
-    M(SettingUInt64, local_tunnel_version, 2, "1: not refined, 2: refined")
+    M(SettingUInt64, local_tunnel_version, 2, "1: not refined, 2: refined")                                                                                                                                                             \
+    M(SettingUInt64, async_recv_version, 2, "1: reactor mode, 2: no additional threads")                                                                                                                                                \
+    M(SettingUInt64, recv_queue_size, 0, "size of ExchangeReceiver queue, 0 means the size is set to tiflash_num * 50")
```

Comment:
`tiflash_num` makes me a little confused, maybe `data_source_mpp_task_num` is better?

Reply:
> `tiflash_num` makes me a little confused, maybe `data_source_mpp_task_num` is better?

okk, I will change it.